.github/workflows/build_and_test.yml

```diff
@@ -761,6 +761,9 @@ jobs:
       run: ./dev/lint-r
     - name: Run documentation build
       run: |
+        # Build docs first with SKIP_API to ensure they are buildable without requiring any
+        # language docs to be built beforehand.
+        cd docs; SKIP_API=1 bundle exec jekyll build; cd ..
```

Comment:
LGTM but just to confirm this doesn't take a lot of time right?

Reply:
Correct. This adds around 5 seconds to the "Run documentation build" step.